prisma/schema.prisma

```diff
@@ -66,7 +66,7 @@ model VerificationToken {
 }
 
 model Source {
-  id              Int    @id @default(autoincrement())
+  id              String @id @default(cuid())
```

Comment:
Everything else in this file is now `cuid`, so I followed the proverbial lemmings on this one.